src/index.js

```diff
@@ -90,17 +93,20 @@ function createSolutionInfos() {
   ));
 }
 
-function processNext(next, config) {
-  const branch = config.submodules[next];
-  console.log(`Processing ${next}`);
+function processSubmodule(name, config) {
+  const branch = config.submodules[name];
 
-  const [owner, repo] = next.split('/');
+  const [owner, repo] = name.split('/');
   const submodule = {
+    _logs: [],
     owner,
     repo,
     branch,
-    remoteUrl: `git@git:${owner}/${repo}.git`
+    remoteUrl: `git@git:${owner}/${repo}.git`,
+    log(message, indent=0) { this._logs = this._logs.concat(message.split(/\r?\n/).map((line) => { return Array(indent + 1).join(' ') + line; })); },
```

Comment:
[String.prototype.repeat()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/repeat)

As in
```js
line => ' '.repeat(indent + 1) + line
```